thumbor/handlers/__init__.py

```diff
@@ -281,7 +281,7 @@ def finish_request(self, context, result_from_storage=None):
             context.config.RESULT_STORAGE_STORES_UNSAFE or not context.request.unsafe)
 
         def inner(future):
-            results, content_type = future.result()
+            results, content_type = (future.result()[0], BaseEngine.get_mimetype(future.result()[0]))
```

Comment:
My only semi-strong opinion is that we should probably only set it once. So if it stays in the `inner` of `finish_request` we should remove https://github.com/okor/thumbor/blob/master/thumbor/handlers/__init__.py#L244 since it will be overridden anyways. 

Also, @guilhermef, not sure what you mean by pointing out that this runs in a thread pool ... seems like both the `finish_request` and `_load_result` execute within the context of the threadpool https://github.com/okor/thumbor/blob/master/thumbor/handlers/__init__.py#L290-L291 Is there some kind of thread safety or race condition concern here?

@guilhermef @heynemann You two have so much more experience with Thumbor code. I defer the choice of where to perform the content type setting to you two.

Also, I just realized that I'll need to add `jpeg2000` to the get_mimetype method, so the jp2 plugin will work as expected. This is the downside of setting the mimetype in one place in thumbor core. Then again, it's rare that we need to add a new MIME type so maybe not a big deal.